src/Maestro/tests/DependencyUpdater.Tests/CheckSubscriptionsAsyncTests.cs

```diff
@@ -25,18 +25,18 @@ public async Task NeedsUpdateSubscription()
             };
             var oldBuild = new Build
             {
-                Branch = "source.branch",
-                Repository = "source.repo",
-                BuildNumber = "old.build.number",
+                AzureDevOpsBranch = "source.branch",
```

Comment:
What's the context for the test changes?

Reply:
This test project was not a part of the solution. When I added it, there were a lot of conflicts as "Branch", "Repository", and "BuildNumber" were not valid fields. This change was made to get the tests to compile. This whole test project probably needs some refactoring.

Reply:
I see.